app/parser/v0_0_1/schema_parser.py

```diff
@@ -74,6 +74,7 @@ def __init__(self, schema):
         self.question_factory.register_all({
             'GENERAL': GeneralQuestion,
             'DATERANGE': DateRangeQuestion,
+            'REPEATINGANSWER': GeneralQuestion,
```

Comment:
@ajmaddaford I started doing this, but it doesn't feel like there's much benefit to splitting it out into it's own class. There's very little being done by the Question schema item class. so having another class which also doesn't do very much doesn't make much sense. What would be more useful would be to have a specific StateQuestion type for the repeating answer question, something like `RepeatingAnswerStateQuestion`, this would need to be created during the `construct_state` phase.